Tasks/DownloadPackageV1/package-lock.json

```diff
@@ -1,1100 +1,8 @@
 {
   "name": "downloadpackage",
   "version": "1.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Lockfile downgraded from v2 to v1? Should this be checked in?

Reply:
I have no idea why this happened... but we should be checking in the package-lock file. Will check again why this happened, good catch

Reply:
was using an older npm and node version... updated